test/tracking/test_planar_tracking.py

```diff
@@ -38,11 +38,11 @@ def test_real(self, device, dtype, data):
             if isinstance(data[k], torch.Tensor):
                 data[k] = data[k].to(device, dtype)
         h0, w0 = data["image0"].shape[2:]
-        data["image0"] = resize(data["image0"], (int(h0 // 2), int(w0 // 2)))
-        data["image1"] = resize(data["image1"], (int(h0 // 2), int(w0 // 2)))
+        data["image0"] = rescale(data["image0"], 0.5, interpolation='bilinear', align_corners=False)
+        data["image1"] = rescale(data["image1"], 0.5, interpolation='bilinear', align_corners=False)
         with torch.no_grad():
             tracker.set_target(data["image0"])
-            torch.random.manual_seed(0)
+            torch.manual_seed(0)  # issue kornia#2027
```

Comment:
continue to fail on macos https://github.com/kornia/kornia/actions/runs/3639450296/jobs/6142845358#step:4:9583